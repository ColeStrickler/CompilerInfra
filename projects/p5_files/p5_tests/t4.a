a:int;
b:bool;
f : () -> void {
	
}

g : () -> void {
	toconsole f();
}
