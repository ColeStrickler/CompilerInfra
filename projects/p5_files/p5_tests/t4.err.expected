FATAL [8,12]-[8,15]: Attempt to output void
Type Analysis Failed
